GVFS/GVFS.Platform.Windows/ProjFSFilter.cs

```diff
@@ -251,13 +253,6 @@ public static bool TryEnableAutoLogger(ITracer tracer)
                     return true;
                 }
 

```

Comment:
Is the method name - `TryEnableOrInstallDriver` - still accurate? I don't think it tries to install the driver anymore.

Reply:
> Is the method name - TryEnableOrInstallDriver - still accurate? I don't think it tries to install the driver anymore.

Yep this is still accurate, we will install the driver when ProjFS is not inbox (see line 259):

`return TryInstallProjFSViaINF(tracer, fileSystem);`

Reply:
ah - that was below the fold, and I missed it. Thanks!